src/helpers/createMethods.ts

```diff
@@ -12,6 +12,10 @@ function getFieldDefinition(
   const fakeLine = docLines.find((line) => line.startsWith('FAKE:'));
   const fakeValue = fakeLine?.replace('FAKE:', '');
 
+  if (fakeValue) {
```

Comment:
I wonder if we should remove the code related to JSON now that we support a simpler approach that is valid for all fields.. what are your thoughts on it?
https://github.com/luisrudge/prisma-generator-fake-data/pull/20/files#diff-9f2dd6068c2dc5460ec9d76b011ed01f9148e427b807ddb5b96d03f259253f97R140-R154

Reply:
> I wonder if we should remove the code related to JSON now that we support a simpler approach that is valid for all fields.. what are your thoughts on this? https://github.com/luisrudge/prisma-generator-fake-data/pull/20/files#diff-9f2dd6068c2dc5460ec9d76b011ed01f9148e427b807ddb5b96d03f259253f97R140-R154
> 
> seems like this code will never run with this change, so we might as well remove it.

I left it because I saw the `logger.warn()` call but now I see that it only triggers if the comment is "empty" i.e. just `///FAKE:`. I removed the code and added a similar check to the top and changed the wording. Also, maybe I'm doing something wrong, but I can't get `logger.warn()` to print anything when running `prisma generate`, I only see log messages when running the tests.